app/views/curation_concerns/base/_show_actions.html.erb

```diff
@@ -1,7 +1,10 @@
 <div class="form-actions">
   <% if Sufia.config.analytics %>
-    <%= link_to "Analytics", sufia.stats_work_path(@presenter), id: 'stats', class: 'btn btn-default' %>
-  <% end %>
+    <% if @presenter.model_name.name ==  "GenericWork" %>
+      <%= link_to "Analytics", sufia.stats_work_path(@presenter), id: 'stats', class: 'btn btn-default' %>
```

Comment:
@njaffer Oh, right. Derp. Below is what I should have typed.

Let's put the stats path code in the presenters, like so:

```ruby
# WorkShowPresenter
def stats_path
    sufia.stats_work_path(self)
end
```

and 

```ruby
# FileSetPresenter
def stats_path
    sufia.stats_file_path(self)
end
```